helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixAdmin.java

```diff
@@ -406,7 +407,41 @@ public ZNRecord update(ZNRecord currentData) {
     }
   }
 
+
   @Override
+  public boolean isEvacuateFinished(String clusterName, String instanceName) {
+    HelixDataAccessor accessor =
+        new ZKHelixDataAccessor(clusterName, new ZkBaseDataAccessor<ZNRecord>(_zkClient));
+    PropertyKey.Builder keyBuilder = accessor.keyBuilder();
+
+    // check the instance is alive
+    LiveInstance liveInstance = accessor.getProperty(keyBuilder.liveInstance(instanceName));
+    if (liveInstance == null) {
+      logger.warn("instance {} in cluster {} is not alive.", instanceName, clusterName);
+      return false;
```

Comment:
Let's make it true as it designed for whether we can continue following operation when the evacuated is done.